test/error_storage_test.exs

```diff
@@ -16,7 +16,9 @@ defmodule ErrorStorageTest do
   describe "add_errors/2" do
     test "appends the error to its proper error kind" do
       BoomNotifier.ErrorStorage.add_errors(@error_kind, @error_info)
-      assert %{@error_kind => {1, [@error_info]}} == Agent.get(:boom_notifier, fn state -> state end)
+
+      assert %{@error_kind => {1, [@error_info]}} ==
+               Agent.get(:boom_notifier, fn state -> state end)
```

Comment:
NOTE FOR REVIEWER: changes in this file where due to mix format. Feel free to ignore